src/pages/recipes/egg-fried-rice.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Deep Dive on Egg Fried Rice'
+date: '2021-06-25'
+categories: ['Recipes']
+status: 'published'
+---
+
+Egg Fried Rice (蛋炒饭 dan chao fan) is the measure of a Chinese home cook. When done well it is table-slappingly good.
+
+<!-- excerpt end -->
+
+When it's done poorly I will still eat the whole dang thing doesn't matter. It's an easy and satisfying comfort food that goes with any meal of the day.
+
+There are many variants of this dish, feel free to switch it up. I don't have precise measurements but here's a video of me making it and eating it: https://www.youtube.com/watch?v=jn2r8DQwCLI
+
+_by Lester Han, guest contributer_

```

Comment:
Cannot believe Lester gets creds for this and an immediate merge but https://github.com/groman00/romanwebnet-gatsby/pull/10/files !!!!!!!!!!!!!! does not get merged???????